### ISSUE TITLE
Ensure pinning is made on ceramic nodes after update

### ISSUE THREAD

**Issue by jjperezaguinaga** [hoprnet/hopr-leaderboard#123]
## Expected Behavior

Doing a write for our registered nodes would be pinned by default in the Ceramic network.

## Current Behavior

Pinning needs to be done explicitly despite docs hinting [opts](https://developers.ceramic.network/streamtypes/tile-document/api/#opts-optional_1) this is the default behavior. Turns out we need to [actually pin](https://developers.ceramic.network/build/javascript/pinning/) our documents in the network. The ceramic team already created ceramicnetwork/docs#139 to highlight this in the docs.

## (Optional) Possible Solution

Add the pinning command.

## Steps to Reproduce

1. Register your node in https://network.hoprnet.org/node
2. Be all happy cause you'll get NFTs
3. Come back in a few days and see your node not registered
4. Be all sad now

## Context (Environment)

In short, we were not pinning our records, so the ceramic network would not keep an entry of our nodes' info.
